redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+                if success["success"]:
+                    success_list.append(user)
+
+                    try:
+                        await modlog.create_case(
+                            self.bot,
+                            guild,
+                            ctx.message.created_at.replace(tzinfo=timezone.utc),
+                            "cmute",
+                            user,
+                            author,
+                            reason,
+                            until=until,
+                            channel=channel,
+                        )
+                    except RuntimeError as e:
+                        log.error(_("Error creating modlog case"), exc_info=e)
+                else:
+                    return await ctx.send(success["reason"])
+            if success_list:
+                verb = _("has")
+                if len(success_list) > 1:
+                    verb = _("have")
+                await channel.send(
+                    _("{users} {verb} been muted in this channel{time}.").format(
+                        users=humanize_list([f"{u}" for u in success_list]), verb=verb, time=time
+                    )
+                )
+
+    @commands.command(usage="[users...] [reason]")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_roles=True)
+    async def unmute(
+        self,
+        ctx: commands.Context,
+        users: commands.Greedy[discord.Member],
+        *,
+        reason: Optional[str] = None,
+    ):
+        """Unmute users.
+
+        `[users...]` is a space separated list of usernames, ID's, or mentions.
+        `[reason]` is the reason for the unmute.
+        """
+        if not users:
+            return await ctx.send_help()
+        if ctx.me in users:
+            return await ctx.send(_("You cannot unmute me."))
+        if ctx.author in users:
+            return await ctx.send(_("You cannot unmute yourself."))
+        if not await self._check_for_mute_role(ctx):
+            return
+        async with ctx.typing():
+            guild = ctx.guild
+            author = ctx.author
+            audit_reason = get_audit_reason(author, reason)
+            success_list = []
+            for user in users:
+                success, issue = await self.unmute_user(guild, author, user, audit_reason)
+                if success:
+                    success_list.append(user)
+                    try:
+                        await modlog.create_case(
+                            self.bot,
+                            guild,
+                            ctx.message.created_at.replace(tzinfo=timezone.utc),
+                            "sunmute",
+                            user,
+                            author,
+                            reason,
+                            until=None,
+                        )
+                    except RuntimeError as e:
+                        log.error(_("Error creating modlog case"), exc_info=e)
```

Comment:
Logging should not be translated.
```suggestion
                        log.error("Error creating modlog case", exc_info=e)
```